### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by wsx864321**
Signed-off-by: gemaizi <864321211@qq.com>

### CODE REVIEW

**Comment by wsx864321** [comment]
fix typo,This word should not be plural.please review @beorn7 @roidelapluie   thx

File: tsdb/block.go [hunk 1]
```diff
@@ -40,7 +40,7 @@ import (
 // IndexWriter serializes the index for a block of series data.
 // The methods must be called in the order they are specified in.
 type IndexWriter interface {
-	// AddSymbols registers all string symbols that are encountered in series
+	// AddSymbol registers all string symbols that are encountered in series
```
Comment (beorn7) [inline tsdb/block.go:43]: The method signature used to be `AddSymbols(sym map[string]struct{})`, so I'm afraid we also need to update the content of the doc comment, not just the first word. It appears the current version of the method only adds a single symbol.

**Review by beorn7 — CHANGES_REQUESTED** [review]

**Comment by beorn7** [comment]
Thanks for the catch @wsx864321 .

File: tsdb/block.go (hunk 1 above)
Comment (wsx864321) [reply tsdb/block.go:43]: OK,got,then I'll revise it and submit it again.Thx.

Comment (wsx864321) [reply tsdb/block.go]: It has been revised again. Thank you for your suggestion,please review again. @beorn7

File: tsdb/block.go [hunk 2]
```diff
@@ -40,8 +40,7 @@ import (
 // IndexWriter serializes the index for a block of series data.
 // The methods must be called in the order they are specified in.
 type IndexWriter interface {
-	// AddSymbols registers all string symbols that are encountered in series
-	// and other indices. Symbols must be added in sorted order.
+	// AddSymbol register a single symbol.
```
Comment (beorn7) [inline tsdb/block.go:43]: Grammar nit: register → registers

I believe `AddSymbol` still has to be called in sorted order. If so, that would be nice to include in the doc comment. Similar to the last sentence that was there before. Something like:

`// AddSymbol registers a single symbol. Symbols must be registered in sorted order.`

**Review by beorn7 — CHANGES_REQUESTED** [review]
Sorry for the delay. We were all busy with KubeCon etc.

File: tsdb/block.go (hunk 2 above)
Comment (wsx864321) [reply tsdb/block.go:43]: Indeed, the expected parameters are sorted.It has been revised again. Please review,thx.

**Review by beorn7 — CHANGES_REQUESTED** [review]

File: tsdb/block.go (hunk 2 above)
Comment (wsx864321) [reply tsdb/block.go]: Thank you for your preciseness.It has been revised again.😁

**Review by beorn7 — APPROVED** [review]
Thanks for your patience.

**Comment by wsx864321** [comment]
> Thanks for your patience.

Thx,Thank you for your guidance.😁

**Pull request merged**